pxtarget.json

```diff
@@ -260,6 +260,36 @@
                 }
             }
         },
+        "nrf52833": {
+            "serial": {
+                "useHF2": true
+            },
+            "compile": {
+                "openocdScript": "source [find interface/cmsis-dap.cfg]; source [find target/nrf52.cfg]",
+                "flashChecksumAddr": 0,
+                "webUSB": true,
+                "flashEnd": 1007616,
```

Comment:
doesn't it have 512k of flash?

Reply:
I was unsure how to calculate flashEnd... Presumably for the 840 flashEnd accounts for various reserved flash pages for BLE? Because I was not sure, In the end, I right shifted flashEnd for the nrf52840 :smile: 

Do you know how the original flashEnd calculation was reached for the 52840?